Ports/lolcat/patches/0001-Remove-unnecessary-headers.patch

```diff
@@ -0,0 +1,37 @@
+From 1855230d5734e829bd7569c8329ed2bb17b671d5 Mon Sep 17 00:00:00 2001
+From: Ninad Sachania <ninad.sachania@gmail.com>
+Date: Wed, 13 Nov 2024 08:46:33 +0000
+Subject: [PATCH] Remove unnecessary headers
```

Comment:
Want to try to get bring this upstream? Looks like it's truly not needed.

(err.h is a BSD-ism that declares `err verr errc verrc errx verrx warn vwarn warnc vwarnc warnx vwarnx err_set_file err_set_exit err_set_exit_b`, none of which are used.)

Fine as-is in the meantime – this is consistent with  Ports/imgcat/patches/0001-Remove-an-include-of-err.h.patch

Reply:
Yes, I'll make make a PR upstream.

Reply:
I made a PR to get this patch merged upstream. Let's see if it gets merged.

Link: https://github.com/jaseg/lolcat/pull/60